school.go

```diff
@@ -0,0 +1,28 @@
+package gofakeit
+
+import "math/rand"
+
+// School will generate a random School type
+func School() string { return school(globalFaker.Rand, globalFaker.Rand, globalFaker.Rand) }
+
+func (f *Faker) School() string { return school(f.Rand, f.Rand, f.Rand) }
+
+func school(r *rand.Rand, r1 *rand.Rand, r2 *rand.Rand) string {

```

Comment:
Only one rand please

Reply:
I've also fixed that but I have a question. globalFaker creates a rand value everytime it is created I suppose, am I right? So wouldn't it be better to create more than one rand value, instead of using just one value all the time?